src/docraptor.ts

```diff
@@ -45,12 +44,11 @@ export async function generatePdfDocRaptor(options: DocRaptorGenerateOptions) {
     body: JSON.stringify(body)
   });
 
-
   if (!response.ok) {
-    throw new Error('Failed to generate PDF: ' + response.statusText + "\n" + await response.text());
+    throw new Error('Failed to generate PDF: ' + response.statusText + '\n' + (await response.text()));
   }
 
-  return new PdfResult(await response.buffer());
+  return new PdfResult(Buffer.from(await response.arrayBuffer()));
```

Comment:
Please create a helper function that will do this `Buffer.from(...)` under the hood. It can be imported as `fetch` and the response be used like in the past `await response.buffer()`